### PR TITLE
EG-608 adding end point information for ocelot

### PR DESCRIPTION
**Pull request by LesWilson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,158 @@ This repo models a process allowing external guidance authors, approvers and pub
 ### License
 
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+
+## Endpoints
+
+### `POST: /external-guidance/scratch`
```
Comment (woollett-r) [inline README.md:13]: /external-guidance/process/scratch

File: README.md [hunk 2]
```diff
@@ -7,3 +7,158 @@ This repo models a process allowing external guidance authors, approvers and pub
...
+
+#### Request Body
+
+This is the json string representing the Ocelot process
+
+#### Success Response
+
+**HTTP Status**: `201`
+
+**Example Response Body**:
```
Comment (macgrewal) [inline README.md:28]: Please can you add something around the `Location` header as this is what they need to redirect the user to so they can view what was pushed up.

File: README.md [hunk 3]
```diff
@@ -7,3 +7,158 @@ This repo models a process allowing external guidance authors, approvers and pub
...
+            <td><pre>BAD_REQUEST</pre></td>
+        </tr>
+        <tr>
+            <td><pre>500</pre></td>
+            <td><pre>INTERNAL_SERVER_ERROR</pre></td>
+        </tr>
+    </tbody>
+</table>
+
+### `POST: /external-guidance/approval/fact-check`
```
Comment (woollett-r) [inline README.md:64]: /external-guidance/process/approval/fact-check

File: README.md (hunk 1 above)
Comment (LesWilson) [reply README.md:13]: Thanks - I started updating wrong README and copied external-guidance scratch URL :(

File: README.md (hunk 2 above)
Comment (LesWilson) [reply README.md:28]: updated, will push shortly

File: README.md (hunk 3 above)
Comment (LesWilson) [reply README.md:64]: thanks - same issue I had with scratch

**Pull request merged**